### DISCUSSION TITLE
Deep linking / Web url prefix stack push

### DISCUSSION THREAD

**Discussion by Ortes** [Milad-Akarie/auto_route_library#1935] [Q&A] [answered]
I build an app for the web platform and I can't make the stack initialize correctly when entering the url in the browser.
I have 2 routes `/` => is a home with a list of groups and a `/groups/:groupId` to get groups details. I do not know if I am supposed to make a nested route by filling the `children`  of my initial route or a just list them one after the other ?
I need the `/` to be added to the stack when /groups/id is entered in the url is this possible ?

**Comment by Ortes** [comment, marked as answer]
I found a solution by adding `includePrefixMatches` to true in the router root config but the documentation said that the default value was true but it is not in the version `7.8.4`

**Comment by Milad-Akarie** [comment]
@Ortes that's an old fact, will change in next push
